src/commands.cpp

```diff
@@ -1959,50 +1959,87 @@ void CommandUserRequest::procresult()
     client->app->invite_result(e);
 }
 
-CommandPutUA::CommandPutUA(MegaClient* client, const char *an, const byte* av, unsigned avl)
+CommandPutUA::CommandPutUA(MegaClient* client, const char *an, const byte* av, unsigned avl, int ctag)
 {
-    cmd("up");
+    this->an = an;
+    this->av.assign((const char*)av, avl);
+    this->u = client->ownuser();
 
-    // if removing avatar, do not Base64 encode the attribute value
-    if (!strcmp(an, "+a") && !strcmp((const char *)av, "none"))
-    {
-        arg(an,(const char *)av, avl);
-    }
-    else
+    cmd("upv");
+
+    beginarray(an);
```

Comment:
The new syntax allows to send several attribute values at once, but SDK doesn't implement such a capability.
Should we add it? It would be useful for very couple attributes, like setting `keyring` + `puCu255` + `puEd255` all at once. It will make the request fail if any of the values cannot be set (ie. outdated version of any of them).